draft-ietf-ppm-dap.md

```diff
@@ -1765,14 +1817,17 @@ Otherwise, the Leader proceeds as follows with each report:
 
 1. Else if the type is "finished" and `state == Finished(out_share)`, then
    preparation is complete and the Leader stores the output share for use in
-   the collection flow ({{collect-flow}}).
+   the collection interaction ({{collect-flow}}).
 
 1. Else if the type is "reject", then the Leader rejects the report and removes
    it from the candidate set.
 
 1. Else the type is invalid, in which case the Leader MUST abort the
    aggregation job.
 
+When the Leader stores the `out_share`, it MUST also store the report ID for
```

Comment:
editorial: I think this text would fit better in the numbered list handling the `Finished(out_share)` case.

Reply:
ack, won't fix. (I don't think that flows works, but we can revisit once the semantics is right.)

Reply:
Ok, having fixed everything else, I think what you're describing works but is repetitive. I'd rather not have to repeat "MUST store the report ID for replay protection" more times than we have to. But if you feel strongly about this I can fix.